automation/build-bin.ts

```diff
@@ -354,6 +370,17 @@ export async function buildOclifInstaller() {
 	let packOpts = ['-r', ROOT];
 	if (process.platform === 'darwin') {
 		packOS = 'macos';
+		await new Promise((resolve, reject) => {
+			klaw('node_modules/node-unzip-2/testData/')
```

Comment:
This will miss most of the zip files:
```
-rw-r--r--  1 0      0        1636 Mar  2 10:19 ./node_modules/unzip-stream/testData/compressed-flags-set/archive.zip
-rw-r--r--  1 0      0        1598 Mar  2 10:19 ./node_modules/unzip-stream/testData/compressed-OSX-Finder/archive.zip
-rw-r--r--  1 0      0        1636 Mar  2 10:19 ./node_modules/unzip-stream/testData/compressed-standard/archive.zip
...
```
To be more "robust", I'd suggest navigating the whole of `node_modules`, look for folders with the word "test" in the name (case insensitive) and delete any zip files under those folders. It should be fairly safe to delete zip files that used in test code.